hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/ProcessFailureExitHandler.java

```diff
@@ -0,0 +1,13 @@
+package org.apache.mesos.hdfs.executor;
+
+import org.apache.mesos.hdfs.util.HDFSConstants;
+import org.apache.mesos.hdfs.util.FailureUtils;
+
+/**
+ * When a process fails this handler will throw a runtime exception.
```

Comment:
Oh yeah, I changed the comment when I was trying to adhere to the findbugs suggestion that I throw a runtime exception instead of exiting.  I'll fix it.